packages/graphql/README.md

```diff
@@ -281,6 +281,18 @@ final result = await graphQLClient.mutate(
 );
 ```
 
+*CSRF Error:* If you are receiving csrf error from your apollo graphql server while uploading file,
+your need to add some additional headers to the `HttpLink`:
+<br>
+Also ensure that you're add `contentType` to `MultipartFile` as `MediaType`
+
+```dart
+HttpLink httpLink = HttpLink('https://api.url/graphql', defaultHeaders: {
+  'Content-Type': 'application/json; charset=utf-8',
+  'X-Apollo-Operation-Name': 'post'
+})
+```
+
```

Comment:
We can put this in section # Q&A  with the following format
```suggestion
## Q&A

*CSRF Error:* If you are receiving csrf error from your apollo graphql server while uploading file,
your need to add some additional headers to the `HttpLink`:
<br>
Also ensure that you're add `contentType` to `MultipartFile` as `MediaType`

```dart
HttpLink httpLink = HttpLink('https://api.url/graphql', defaultHeaders: {
  'Content-Type': 'application/json; charset=utf-8',
  'X-Apollo-Operation-Name': 'post'
})
```
- I'm receiving a `CSRF Error` from apollo graphql server while uploading file, what I need to do?
Add some additional headers to the `HttpLink`:
<br>
Also ensure that you're add `contentType` to `MultipartFile` as `MediaType`

```dart
HttpLink httpLink = HttpLink('https://api.url/graphql', defaultHeaders: {
  'Content-Type': 'application/json; charset=utf-8',
  'X-Apollo-Operation-Name': 'post'
})
```
```